lib/fibered_mysql2/hash_config_override.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module FiberedMysql2
+  module HashConfigOverride
+    # Override the reaping_frequency method to return nil so that the connection pool does not reap connections when in fibered mode.
+    def reaping_frequency
+      nil
+    end
+  end
+end
+
+if ActiveRecord.gem_version >= "6.1"
+  ActiveRecord::DatabaseConfigurations::HashConfig.prepend(FiberedMysql2::HashConfigOverride)
```

Comment:
Can you add a spec that confirms this override has the desired effect of returning `nil` even when the reaping frequency is configured?

(I get that it might have been necessary to add this to get 6.1 to run specs at all, but still it would be best to have a spec that would fail if this override stopped working for any reason.)

Reply:
Added: https://github.com/Invoca/fibered_mysql2/pull/12/commits/d557dea3e19c1653ab83e381b360e01e2e81a2d7